intranet-webapp/MediaLibrary.Intranet.Web/Controllers/GalleryController.cs

```diff
@@ -36,8 +36,14 @@ public async Task<IActionResult> Item([BindRequired, FromRoute] string id)
             // Get item info and check if user is author
             bool isAuthor = (await GetItemAuthorAsync(id)) == User.GetUserGraphEmail();
 
+            // Get item upload date info and check if within 1 day
+            System.DateTime itemUploadDateTime = (await GetItemUploadDateAsync(id)).ToLocalTime();
+            System.DateTime currentDateTime = System.DateTime.Now.ToLocalTime();
```

Comment:
`System.DateTime.Now` is already in local time, so calling ToLocalTime() isn't strictly needed

But as a general rule, it is generally preferred to [use UTC when working with DateTime](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime#the-datetime-structure):

```suggestion
            System.DateTime itemUploadDateTime = await GetItemUploadDateAsync(id);
            System.DateTime currentDateTime = System.DateTime.UtcNow;
```